### PR TITLE
adding new package icon

### PR DESCRIPTION
**Pull request by msft-ray**
Added the following element to DocumentFormat.OpenXml.Package.Props:

`<PackageIconUrl>https://raw.githubusercontent.com/OfficeDev/Open-XML-SDK/master/icon.png</PackageIconUrl>`

Added the vector graphic source file and the png for the icon.

- icon.svg
- icon.png

### CODE REVIEW

**Comment by twsouthwick** [comment]
What's the resolution of png? It needs to be 64x64 for NuGet

**Comment by msft-ray** [comment]
@twsouthwick the png is precisely 64 pixels by 64 pixels.

File: icon.svg [hunk 1]
```diff
@@ -0,0 +1,663 @@
...
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="64"
+   height="64"
+   viewBox="0 0 16.933333 16.933334"
+   version="1.1"
+   id="svg11519"
+   inkscape:version="0.92.2 (5c3e80d, 2017-08-06)"
+   sodipodi:docname="ooxml_sdk_icon_20.svg"
+   inkscape:export-filename="D:\OneDrive - Microsoft\Open XML SDK\ooxml icons\ooxml_sdk_icon_20.png"
```
Comment (twsouthwick) [inline icon.svg]: Do you want this in there?

Comment (msft-ray) [reply icon.svg]: Good catch, @twsouthwick. No, I don't want that in there. I updated it with a clean svg.

**Pull request merged**